src/System.Private.CoreLib/shared/System/Globalization/CalendarData.Windows.cs

```diff
@@ -56,11 +57,11 @@ private bool LoadCalendarDataFromSystem(string localeName, CalendarId calendarId
 
             // String Arrays
             // Formats
-            ret &= CallEnumCalendarInfo(localeName, calendarId, CAL_SSHORTDATE, LOCALE_SSHORTDATE | useOverrides, out this.saShortDates);
```

Comment:
the reason those are `!` is because this is still part of the initialization and further down we make sure they are never null (changing them to nullable will cause that we will have to add hacks all over the place - ideal solution would be to refactor this class)